scripts/oncotree_to_oncotree.py

```diff
@@ -65,23 +66,23 @@ def validate_input_oncotree_versions(oncotree_versions_list, source_version, tar
         sys.exit(1)
 
 #--------------------------------------------------------------
-def validate_and_fetch_oncotree_version_indexes(source_version, target_version, oncotree_api_url_base):
-    if source_oncotree_version_name == target_oncotree_version_name:
+def validate_and_fetch_oncotree_version_release_dates(source_version, target_version, oncotree_api_url_base):
+    if source_version == target_version:
         print >> sys.stderr, "Error: Source oncotree version (%s) and target oncotree version (%s) are the same.  There is no need to convert this file." % (source_version, target_version)
     oncotree_versions_list = fetch_oncotree_versions(oncotree_api_url_base)
 
     # validate source and target versions
     validate_input_oncotree_versions(oncotree_versions_list, source_version, target_version)
 
-    # return indexes of source and target versions from available oncotree versions
-    source_oncotree_version_index, target_oncotree_version_index = -1, -1
-    for index, version in enumerate(sorted(response.json(), key = lambda k: k[VERSION_RELEASE_DATE_FIELD])):
-        if version[VERSION_API_IDENTIFIER_FIELD] == source_oncotree_version_name:
-
-            source_oncotree_version_index = index
+    # return release dates of source and target versions from available oncotree versions
+    source_oncotree_version_release_date = -1
+    target_oncotree_version_release_date = -1
```

Comment:
@sheridancbio i think the dates should be parsed as `datetime` objects. Here, these variables are set as `-1`, which is an integer, but then they get set as some string value below.

I think default should be `None`

and that we should parse the `version[VERSION_RELEASE_DATE_FIELD]` as `datetime` objects. 

Once parsed we can easily make the comparison to determine the directionality assuming that neither `source_version_date` or `target_version_date` are `None`.